crates/counter-agg/src/lib.rs

```diff
@@ -96,23 +116,18 @@ impl CounterSummary {
     }
 
     fn single_value(&self) -> bool {
```

Comment:
@davidkohn88 I'd particularly appreciate your thoughts on this change.

last == first does not mean single value on a gauge.  However, self.stats.n is only 1 when there is a single value if I'm reading it correctly. I added a test for gauge that fails in the expected manner without this fix. I can make existing counter tests fail by having this method return either true or false unconditionally, which suggests the existing tests do demonstrate this works.

Reply:
I think `last == first` is still a valid test on a gauge.  First and last are `TSPoint`s which is a value/timestamp pair, so even if the value ends back where it started, the timestamp should be different.  That being said, `self.stats.n` might be a more direct check.

Reply:
Yep the original check was fine! I reverted this change.